src/lib/markbind/src/parser.js

```diff
@@ -70,7 +57,7 @@ class Parser {
         return;
       }
       const resourcePath = utils.ensurePosix(elem.attr('src'));
-      if (utils.isAbsolutePath(resourcePath) || utils.isUrl(resourcePath)) {
+      if (path.isAbsolute(resourcePath) || utils.isUrl(resourcePath)) {
```

Comment:
previously `utils.isAbsolutePath` also checks for the existence of the `{{ baseUrl }}` substring. This is no longer necessary, so we could call `path.isAbsolute` directly